NAudio.Midi/Midi/MidiInSysexMessageEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+
+namespace NAudio.Midi
+{
+    /// <summary>
+    /// MIDI In Sysex Message Information
+    /// </summary>
+    public class MidiInSysexMessageEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Create a new Sysex MIDI In Message EventArgs
+        /// </summary>
+        /// <param name="message"></param>
```

Comment:
use correct parameter name

Reply:
Corrected - with a simple explanation of its purpose (obvious also from the proper name)